asdf_astropy/converters/coordinates/tests/test_masked.py

```diff
@@ -0,0 +1,74 @@
+import asdf
+import numpy as np
+import pytest
+from astropy import units as u
+from astropy.coordinates import Angle, EarthLocation, Latitude, Longitude, SkyCoord
+from astropy.utils.masked import Masked
+
+from asdf_astropy.testing.helpers import skip_if_astropy_lt_7_1
+
```

Comment:
```suggestion
pytest.skip(ASTROPY_LT_7_1, allow_module_level=True)
```
Let's just skip this whole file is the astropy version isn't satisfied (we will need to find a suitable place for ASTROPY_LT_7_1) that way we can remove the `skip_if_astropy_lt_7_1` decorator.

Reply:
I don't think that `pytest.skip` takes a boolean option. You're thinking of `pytest.mark.skipif`. But I did something similar; please take a look.

Reply:
Thanks. Let's go with:
```python
if ASTROPY_LT_7_1:
    pytest.skip("These tests require astropy >= 7.1dev", allow_module_level=True)
```
to allow removal of `skip_if_astropy_lt_7_1`.

Reply:
Done.